### ISSUE TITLE
Separate liver and tumor label at the same time

### ISSUE THREAD

**Issue by NBIronman** [davidiommi/SALMON---3D-Medical-Images-Segmentation#2]
Hello, I want to segment the liver and tumor labels in LITS at the same time. Can they be used directly? Or need to modify the number of channels in the code?

**Comment by NBIronman** [comment]
Another problem is that when I run train, the Loading dataset will always be loaded after a short period of time. I found that my memory usage has reached the upper limit. How can I modify it to reduce the memory usage?

**Comment by davidiommi** [comment]
Hi, sorry for the late reply. Yes, you can do multi-label segmentation. Look at this tutorial: https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb You have to set two channels out in the transforms. and change the loss function. 

Memory usage is either with smaller batch_size, patch_size, or instead of : check_train = "monai.data.CacheDataset(data=train_dicts, transform=train_transforms)" use "monai.data.Dataset"

**Comment by davidiommi** [comment]
Hi, I added the modified version for multi-label segmentation. If you have liver and tumor remember to put 3 channels output, because you must include also the background.

For the dataset I confirm you must change Cache to Dataset in the loader
